.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -9,20 +9,32 @@ body:
 
         For help or advice on using Python, try one of the following options instead of opening a GitHub issue:
 
-          - Posting on [Discourse](https://discuss.python.org/c/users/7)
+          - Asking on [Discourse](https://discuss.python.org/c/users/7) or [Stack Overflow](https://stackoverflow.com)
           - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
           - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
-  - type: checkboxes
+
+        **Is this issue new?**
+
+        Make sure to search the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc) to check that the bug has not already been reported.
+
+        **Is this a bug in CPython?**
+
+        Only use this form to report bugs in the CPython project itself. Bugs in third-party projects (e.g. ``pip`` or ``requests``) should be reported to that project instead.
```

Comment:
Hmm, I'm not sure about this @AA-Turner... I worry that the longer we make this MarkDown block at the beginning of the issue template, the less likely people are to read any of it. Three bullet-points at the beginning isn't too bad, but three sub-paragraphs feels like an information overload for issue reporters: https://github.com/AlexWaygood/cpython/issues/new?assignees=&labels=type-bug&projects=&template=bug.yml

Reply:
I'd bin "Is this a bug in CPython?" first

Reply:
Adding the heading for "Is this issue new?" was mainly as I felt it was a non-sequitur under "New to Python?". But perhaps it doesn't matter.

Reply:
yeah, the non-sequitur issue was why I originally started the sentence with "Also". Perhaps we could compromise on:

> Make sure to also search the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc) to check that the bug has not already been reported.

(It's a split infinitive, but who really cares about split infinitives? 😛)

Reply:
> This is true, but if you're somebody who's been using Python for a while, you're also more likely to know that you should probably check the issue tracker first before reporting an issue.

My scenario was more someone who is used to *using* Python, and might've contributed to some other part of the Python world, but is newer to contributing to CPython. I think you're right about capturing the many, though.